cdap-watchdog/src/main/java/co/cask/cdap/metrics/data/TimeSeriesTable.java

```diff
@@ -369,20 +512,8 @@ private static void put(NavigableMap<byte[], NavigableMap<byte[], byte[]>> table
                         entityCodec.encode(MetricsEntityType.RUN, runId));
   }
 
-  private byte[] getPaddedKey(String contextPrefix, String runId, String metricPrefix, String tagPrefix,
-                              int timeBase, int padding) {
-
-    // If there is no contextPrefix, metricPrefix or runId, just applies the padding
-    return Bytes.concat(
-      entityCodec.paddedEncode(MetricsEntityType.CONTEXT, contextPrefix, padding),
-      entityCodec.paddedEncode(MetricsEntityType.METRIC, metricPrefix, padding),
-      entityCodec.paddedEncode(MetricsEntityType.TAG, tagPrefix, padding),
-      Bytes.toBytes(timeBase),
-      entityCodec.paddedEncode(MetricsEntityType.RUN, runId, padding));
-  }
-
   private FuzzyRowFilter getFilter(MetricsScanQuery query, long startTimeBase,
-                                   long endTimeBase, boolean shouldMatchAllTags) {
+                                   long endTimeBase, boolean shouldMatchAllTags, byte[] timeBaseMatch) {
```

Comment:
after discussion: for no time range specified case one fuzzy rule is needed:
```
      fuzzyPairs.add(ImmutablePair.of(Bytes.concat(contextPair.getFirst(), metricPair.getFirst(), tagPair.getFirst(),
                                                   // fill time bytes with zeroes
                                                   FOUR_ZERO_BYTES, runIdPair.getFirst()),
                                      Bytes.concat(contextPair.getSecond(), metricPair.getSecond(), tagPair.getSecond(),
                                                   // FOUR_ONE_BYTES means anything in time bytes
                                                   FOUR_ONE_BYTES, runIdPair.getSecond())));
```
refactor